gui/default/index.html

```diff
@@ -437,6 +437,12 @@ <h4 class="panel-title">
           </div>
         </div>
         <span class="pull-right">
+          <button type="button" class="btn btn-sm btn-default" ng-click="pauseAllFolders()">
+            <span class="fa fa-refresh"></span>&nbsp;<span translate>Pause All Folders</span>
+          </button>
+          <button type="button" class="btn btn-sm btn-default" ng-click="resumeAllFolders()">
```

Comment:
Just setAllFolderPause(false) and setAllFolderPause(true) in the other one, and remove the two functions.